packages/patterns/src/components/category-selector.js

```diff
@@ -64,17 +51,15 @@ export default function CategorySelector( { values, onChange } ) {
 	}
 
 	return (
-		<>
-			<FormTokenField
-				className="patterns-menu-items__convert-modal-categories"
-				value={ values }
-				suggestions={ suggestions }
-				onChange={ handleChange }
-				onInputChange={ debouncedSearch }
-				maxSuggestions={ MAX_TERMS_SUGGESTIONS }
-				label={ __( 'Categories' ) }
-				tokenizeOnBlur={ true }
-			/>
-		</>
+		<FormTokenField
+			className="patterns-menu-items__convert-modal-categories"
+			value={ categoryTerms }
+			suggestions={ suggestions }
+			onChange={ handleChange }
+			onInputChange={ debouncedSearch }
+			label={ __( 'Categories' ) }
+			tokenizeOnBlur={ true }
+			__experimentalExpandOnFocus={ true }
```

Comment:
Nit picking and not important at all :P  Just noting that in JSX `tokenizeOnBlur` is equivalent to `tokenizeOnBlur={ true }`. I usually use the former for conciseness

Reply:
yeh, good point, have tidied that up.